src/_data/docs.yml

```diff
@@ -87,7 +87,7 @@ sidebar:
       items:
         - title: Overview
           href: dependent_images.html
-        - title: Personal Registry
+        - title: Choosing an Image Registry
```

Comment:
Imo this guide is not quite about choosing the registry, since in some cases that choice has already been made for you.
I would change the title to something like "Setting up any Image Registry with Tilt"

Reply:
:+1: